Assets/Scripts/UI/Canvas/StateMachine/UIStateManager.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using UI.Canvas.Introduction;
+using UI.Canvas.PhotoBookSystem;
+using UnityEngine;
```

Comment:
```suggestion
using System;
using UnityEngine;

using UI.Canvas.Introduction;
using UI.Canvas.PhotoBookSystem;
```
[Code convention](https://github.com/Team-Swamp/CaptureGraafsmeer/wiki/Code-convention#importing-namespaces)